### PR TITLE
PPS Alignment - bug fix

### PR DESCRIPTION
**Pull request by MatiXOfficial**
#### PR description:

Fixed a bug from #31728. Now, the PPSAlignmentConfig plugin is registered, and some necessary declarations were added.

Most likely, we will have to backport this update to 12_0.

#### PR validation:

Run the test from CalibPPS/AlignmentGlobal/tests and received expected results.

### CODE REVIEW

File: CondFormats/PPSObjects/interface/PPSAlignmentConfig.h [hunk 1]
```diff
@@ -87,6 +87,8 @@ struct Binning {
 
   unsigned int n_bins_y_;
   double y_min_, y_max_;
+
+  COND_SERIALIZABLE;
```
Comment (francescobrivio) [inline CondFormats/PPSObjects/interface/PPSAlignmentConfig.h:91]: @ggovi usually it is not allowed to change the a class in CondFormat, but this is a bugfix, is it ok?

**Review by francescobrivio — COMMENTED** [review]
@MatiXOfficial I suggest to also add this `CondFormat` to the several utility tools in  `CondCore/Utilites`. You can have a look at https://github.com/cms-sw/cmssw/pull/29662/files to see which utilites might be useful.

**Comment by francescobrivio** [comment]
type bugfix

**Comment by MatiXOfficial** [comment]
@francescobrivio Thanks for the suggestion!

**Comment by francescobrivio** [comment]
@cmsbuild please test

**Comment by francescobrivio** [comment]
@cmsbuild please abort

**Comment by francescobrivio** [comment]
@MatiXOfficial there are still few missing things, see comments by Giacomo https://github.com/cms-sw/cmssw/pull/34845#issuecomment-897530225 and https://github.com/cms-sw/cmssw/pull/34845#issuecomment-897533136

**Comment by francescobrivio** [comment]
@cmsbuild please test

**Comment by francescobrivio** [comment]
+alca

**Comment by ggovi** [comment]
+1

**Comment by perrotta** [comment]
+1

**Pull request merged**